### ISSUE TITLE
[testing-in-a-browser] npm install error

### ISSUE THREAD

**Issue by jcalfee** [bitpay/wallet#11578]
**Describe the bug**
During `npm install` per https://github.com/bitpay/wallet/tree/v12.3.2#testing-in-a-browser there is a compiler error building node-sass.

**To Reproduce**

Happens in both:
```bash
git checkout HEAD
git checkout v12.3.2
```

Steps:
```bash
git clone https://github.com/bitpay/wallet.git
cd wallet
npm install
```

Error:
`../src/create_string.cpp:17:37: error: no matching function for call to 'v8::String::Utf8Value::Utf8Value(v8::Local<v8::Value>&)'`

**Expected behavior**

npm should install

**Desktop (please complete the following information):**

I have tested in Docker `node:13-stretch-slim` with and `node:13-alpine`

Fyi:
Stretch: `apt-get install -y git make python gcc linux-libc-dev linux-headers-$(uname -r) g++`
Alpine: `apk add git python2 ncurses make gcc libc-dev linux-headers eudev-dev g++`

**Additional context**

I realize this is probably a bug caused by minor upgrades or bug fixes in a 3rd party package.  You may however want to consider adjusting your `package.json` or investigating to ensure you can avoid an unexpected breakage during a re-install or in your deployment environment.

If you have more specific information about the node version and OS version that works for you please let me know.

**Comment by moza88** [comment]
Have you been able to resolve this issue? I am coming across the same issue
../src/create_string.cpp:17:25: error: no matching constructor for initialization of 'v8::String::Utf8Value'
  v8::String::Utf8Value string(value);

I am not sure which library is causing this issue.

**Comment by jcalfee** [comment]
No, it is a tricky one.  Unless we can pin-point it I usually fall back to: what version/s are working?

**Comment by iamafresh** [comment]
+1

**Comment by stanfieldr** [comment]
I've been searching the interwebs and debugging because of this issue and thought I'd let you know my findings and in my case it turned out to be related to this issue https://github.com/sass/libsass/issues/2883

Basically I updated `node-sass` to a newer version and everything started working. Not sure in this case
